File: test1.js
#pragma strict

private var _characterMeshIndex = 0;
private var ca : CharacterAssets;

function Start() {
	ca = GameObject.Find("CharacterAssetsManager").GetComponent(CharacterAssets);
	InstantiateCharacterMesh();
	
}

function InstantiateCharacterMesh(){
	var mesh : GameObject = Instantiate(ca.characterMesh[_characterMeshIndex], transform.position, Quarternion.identity) as GameObject ;
}

File: test1.cs
private int characterMeshIndex = 0;
private CharacterAssets ca;

public void Start(){
	ca = GameObject.Find("CharacterAssetsManager").GetComponent(CharacterAssets);
}

public void InstantiateCharacterMesh(){
	GameObject mesh  = (GameObject)(Instantiate(ca.characterMesh[_characterMeshIndex], transform.position, Quarternion.identity))  ;
}